applications/jonny5/src/j5_request.erl

```diff
@@ -124,6 +127,9 @@ request_number(Number, CCVs) ->
             Original
     end.
 
+ccvs(Request) ->
+    Request#request.request_ccvs.
```

Comment:
My personal preference is to pattern match records in the header vs accessing them in the body, if possible. I'm not sure what the performance implications are, if the compiler converts both to the same Core Erlang, or what, but just throwing it out there :)